GooglePlayInstant/Editor/QuickDeploy/QuickDeployConfig.cs

```diff
@@ -23,21 +24,32 @@ namespace GooglePlayInstant.Editor.QuickDeploy
     /// </summary>
     public static class QuickDeployConfig
     {
-        private static readonly string ConfigurationFilePath =
-            Path.Combine("Library", "PlayInstantQuickDeployConfig.json");
+        private static readonly string EditorConfigurationFilePath =
+            Path.Combine("Library", "PlayInstantQuickDeployEditorConfig.json");
+
+        private static readonly string ResourcesDirectoryPath =
+            Path.Combine(LoadingScreenGenerator.SceneDirectoryPath, "Resources");
+
+        private static readonly string EngineConfigurationFilePath =
+            Path.Combine(ResourcesDirectoryPath, LoadingScreenConfig.EngineConfigurationFileName);
 
         /// <summary>
-        /// The Configuration singleton that should be used to read and modify Quick Deploy configuration.
-        /// Modified values are persisted by calling SaveConfiguration.
+        /// The Editor Configuration singleton that should be used to read and modify Quick Deploy configuration.
+        /// Modified values are persisted by calling SaveEditorConfiguration.
         /// </summary>
-        private static readonly Configuration _config = LoadConfiguration();
+        private static readonly EditorConfiguration EditorConfig = LoadEditorConfiguration(EditorConfigurationFilePath);
 
-        public static string CloudCredentialsFileName = _config.cloudCredentialsFileName;
-        public static string AssetBundleFileName = _config.assetBundleFileName;
-        public static string CloudStorageBucketName = _config.cloudStorageBucketName;
-        public static string CloudStorageObjectName = _config.cloudStorageObjectName;
-        public static string AssetBundleUrl = _config.assetBundleUrl;
+        /// <summary>
+        /// The Engine Configuration singleton that should be used to read and modify Loading Screen configuration.
+        /// Modified values are persisted by calling SaveEngineConfiguration.
+        /// </summary>
+        private static readonly LoadingScreenConfig.EngineConfiguration EngineConfig = LoadEngineConfiguration(EngineConfigurationFilePath);
 
+        public static string CloudCredentialsFileName = EditorConfig.cloudCredentialsFileName;
```

Comment:
Please give this block a single "//" comment saying what it is, e.g.

// Copy of fields from EditorConfig and EngineConfig for holding unsaved values set in the UI.